src/main/java/ru/skypro/flea/entity/Ad.java

```diff
@@ -0,0 +1,36 @@
+package ru.skypro.flea.entity;
+
+import lombok.Getter;
```

Comment:
we now have two packages for entities: model and entity
please, choose one of them and join them)
it's not critical, but i think there is no need to keep entities in different packages